dashboard/modules/reporter/reporter_head.py

```diff
@@ -132,7 +133,7 @@ async def get_cluster_status(self, req):
                 message="Got cluster status.",
                 autoscaling_status=legacy_status.decode() if legacy_status else None,
                 autoscaling_error=error.decode() if error else None,
-                cluster_status=formatted_status if formatted_status else None,
```

Comment:
wouldn't we return None if `formatted_status` is falsy?

Reply:
No, we will not enter this branch since frontend call endpoint with query `format=1`, making `return_formatted_output = True`


```
       formatted_status = (
            json.loads(formatted_status_string.decode())
            if formatted_status_string
            else {}
        )

        if not return_formatted_output:
            return dashboard_optional_utils.rest_response(
                success=True,
                message="Got cluster status.",
                autoscaling_status=legacy_status.decode() if legacy_status else None,
                autoscaling_error=error.decode() if error else None,
                cluster_status=formatted_status if formatted_status else "",
            )
```

Reply:
I just make some changes here to prevent we call the endpoint without the parameter in the future. Currently, we do have a query `format=1`